### PR TITLE
Add a Readme.md with usage instructions #17

### PR DESCRIPTION
**Pull request by calbrecht**

### CODE REVIEW

**Comment by Ma27** [comment]
That's definetely a good thing to do. But it would be cool if you could add a header to the readme with a headline, the travis-ci badge and maybe a short sentence what this library basically does :)

**Comment by calbrecht** [comment]
@Ma27 please add a headline to your liking, as i don't have a clue what should be in there.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+[![Build Status](https://travis-ci.org/mayflower/libholiday.svg?branch=master)](https://travis-ci.org/mayflower/libholiday)
+
+A library to calculate holidays.
+
+Supported countries and states
+
+* America
+* Germany
+  * Hamburg, Saarland, BadenWuerttemberg, Hesse, SaxonyAnhalt, Bavaria, Saxony, Berlin, LowerSaxony, SchleswigHolstein, Brandenburg, MecklenburgWesternPomerania, Bremen, Thuringia, NorthRhineWestphalia, RhinelandPalatinate
```
Comment (mmoll) [inline README.md]: I'm a bit unsure if those names should really be in English...

Comment (calbrecht) [reply README.md]: Why not? It gives an international flare and show ambitions :=

**Comment by Ma27** [comment]
@calbrecht I'd be happy with something like `# libholiday`just to give the document some more structure. It seems as I don't have commit access on this repo, so you'd have to do this yourself :)

File: README.md (hunk 1 above)
Comment (Ma27) [reply README.md]: on the other hand, wouldn't it be sufficient to say `Germany and all federal states`?

**Comment by mmoll** [comment]
@Ma27 please check your privilege again...

**Comment by Ma27** [comment]
@calbrecht I've updated the README accordingly.
@mmoll thanks!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Add to your projects ```composer.json```
+
+```json
+{
+    "name": "your/project",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/mayflower/libholiday"
```
Comment (mmoll) [inline README.md]: Do we really want users to use this git repo?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Supported countries and states
+
+* America
+* Germany and all constituent states
+* Sweden
+* Netherlands
+
+Usage
+
+Add to your projects ```composer.json```
```
Comment (mmoll) [inline README.md]: `composer require` is more common now, I think...

Comment (calbrecht) [reply README.md]: yes but how to require from github?

File: README.md (hunk 2 above)
Comment (calbrecht) [reply README.md]: You mean like add it to packagist or the like?

Comment (mmoll) [reply README.md]: Isn't this the repo for https://packagist.org/packages/mayflower/holiday?

Comment (calbrecht) [reply README.md]: You are completely correct. I did only try mayflower/libholiday, damnit.

File: README.md (hunk 3 above)
Comment (calbrecht) [reply README.md]: Ok, done. :+1:

**Comment by calbrecht** [comment]
@mmoll please review the changes :)

**Comment by mmoll** [comment]
merged, danke @calbrecht!

**Pull request merged**